cloud/organization/organization.go

```diff
@@ -115,8 +116,9 @@ func getOrganizationSelection(out io.Writer, coreClient astrocore.CoreClient) (*
 
 func SwitchWithContext(domain string, targetOrg *astrocore.Organization, astroClient astro.Client, coreClient astrocore.CoreClient, out io.Writer) error {
 	c, _ := context.GetCurrentContext()
+
 	// reset org context
-	_ = c.SetOrganizationContext(targetOrg.Id, targetOrg.ShortName)
+	_ = c.SetOrganizationContext(targetOrg.Id, targetOrg.ShortName, fmt.Sprintf("%s", *targetOrg.Product)) //nolint
```

Comment:
Same comment as above, do we need a nil check on `targetOrg.Product` before writing it to context?

Reply:
Added a check